### PR TITLE
Autosave

### PR DESCRIPTION
**Pull request by ethn**
surprise: I'm the guilty party who broke autosave.

(ok, nobody is surprised.)

### CODE REVIEW

File: card/card.gemspec [hunk 1]
```diff
@@ -6,7 +6,7 @@ version = File.open(File.expand_path( '../VERSION', __FILE__ )).read.chomp
 Gem::Specification.new do |s|
   s.name          = 'card'
   s.version       = version
-  s.authors       = ["Ethan McCutchen", "Lewis Hoffman", "Gerry Gleason"]
+  s.authors       = ["Ethan McCutchen", "Lewis Hoffman", "Gerry Gleason", "Philipp Kühl"]
```
Comment (ethn) [inline card/card.gemspec]: as it should be :)

File: card/mod/03_machines/lib/javascript/wagn.js.coffee [hunk 2]
```diff
@@ -101,7 +101,7 @@ jQuery.fn.extend {
   autosave: ->
     slot = @slot()
     return if @attr 'no-autosave'
-    multi = @closest 'fieldset'
+    multi = @closest '.form-group'
```
Comment (ethn) [inline card/mod/03_machines/lib/javascript/wagn.js.coffee]: looks like I broke autosave when I restructured the edit areas for bootstrap.

File: card/mod/06_bootstrap/lib/stylesheets/bootstrap_cards.scss [hunk 3]
```diff
@@ -80,6 +80,18 @@ body.mceContentBody {
...
+.card-slot > .card-header .card-menu-link {
+  display: none;
+}
+.card-slot.closed-view,
+.card-slot:hover {
+  > .card-header .card-menu-link {
+    display: inline-block;
+  }
+}
+
```
Comment (ethn) [inline card/mod/06_bootstrap/lib/stylesheets/bootstrap_cards.scss]: menu shows up when hovering over card.

File: decko-rails/decko-rails.gemspec [hunk 4]
```diff
@@ -5,13 +5,12 @@ version = File.open(File.expand_path( '../../card/VERSION', __FILE__ )).read.cho
 Gem::Specification.new do |s|
   s.name          = 'decko-rails'
   s.version       = version
```
Comment (ethn) [inline decko-rails/decko-rails.gemspec]: I pushed decko-rails spec to rubygems for the first time today (made sure we had the name)

File: card/card.gemspec (hunk 1 above)
Comment (GerryG) [inline card/card.gemspec]: Oh, this is the context.  Welcome to the list of major contributors, nice to have it growing.

File: decko-rails/decko-rails.gemspec (hunk 4 above)
Comment (GerryG) [inline decko-rails/decko-rails.gemspec]: We have decko reserved already, right?

Comment (ethn) [inline decko-rails/decko-rails.gemspec]: yes

**Pull request merged**